templates/ts/action/package.json.tpl

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "{{appName}}",
+    "version": "1.0.0",
+    "description": "Microsoft Teams Toolkit action sample",
+    "engines": {
+        "node": "16 || 18"
+    },
+    "author": "Microsoft",
+    "license": "MIT",
+    "main": "./lib/src/index.js",
+    "scripts": {
+        "dev:teamsfx": "env-cmd --silent -f .localConfigs npm run dev",
+        "dev": "nodemon --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
+        "build": "tsc --build",
+        "start": "node ./lib/src/index.js",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "watch": "nodemon --exec \"npm run start\""
```

Comment:
I heard that this script name should be `watch:teamsfx`, otherwise the watch terminal does not close when local debug ends. Could you help confirm it?